src/test/java/org/folio/rest/api/ItemEffectiveCallNumberTest.java

```diff
@@ -44,53 +44,64 @@ public static void createInstance() throws Exception {
 
   @Test
   public void canInitializeEffectiveCallNumber() throws Exception {
-
     UUID holding = createInstanceAndHoldingWithCallNumber(mainLibraryLocationId);
     Item item = buildItem(holding, null, null);
     EffectiveCallNumberComponents components = new EffectiveCallNumberComponents();
     item.setEffectiveCallNumberComponents(components);
 
-    String query = String.format("INSERT INTO test_tenant_mod_inventory_storage.item (id, jsonb) values ('%s','%s');",
-        item.getId(), mapper.writeValueAsString(item));
+    String query = String.format(
+      "INSERT INTO test_tenant_mod_inventory_storage.item (id, jsonb) values ('%s','%s');",
+      item.getId(), mapper.writeValueAsString(item));
     runSql(query);
 
-    assertThat(getItem(item.getId()).getEffectiveCallNumberComponents().getCallNumber(), is(nullValue()));
+    assertThat(
+      getItem(item.getId()).getEffectiveCallNumberComponents().getCallNumber(),
+      is(nullValue()));
 
     runSql(POPULATE_EFFECTIVE_CALL_NUMBER_SQL);
 
-    assertThat(getItem(item.getId()).getEffectiveCallNumberComponents().getCallNumber(), is("testCallNumber"));
+    assertThat(
+      getItem(item.getId()).getEffectiveCallNumberComponents().getCallNumber(),
+      is("testCallNumber"));
   }
 
   @Test
   public void canInitializeEffectiveCallNumberAfterHoldingsChange() throws Exception {
```

Comment:
Similarly to the other test, I don’t understand why the update is important, as these scripts are to be run on upgrade, where the records are already fixed?

Reply:
Since I have to use sql for any interaction with the database that avoids the business logic it was easier to update an existing item than to create a new valid item. My concern was to show the script updates the value of the components, and an existing one seemed just as good for that.